src/resource/resource-opening-hours/opening-period/PeriodOpeningHours.tsx

```diff
@@ -39,6 +44,40 @@ function renderStartAndEndTimes(
   }`;
 }
 
+function timeSpanDescriptionExistsInSomeLanguage(
+  timeSpanDescriptionObj: LanguageStrings | undefined
+): boolean {
+  if (!timeSpanDescriptionObj) {
+    return false;
+  }
+  return !!(
+    timeSpanDescriptionObj.fi ||
+    timeSpanDescriptionObj.sv ||
+    timeSpanDescriptionObj.en
+  );
+}
+
+function renderTimeSpanDescription(
+  timeSpanDescriptionObj: LanguageStrings | undefined,
```

Comment:
descriptionObject?

Reply:
Since this is only a local variable and more like a helper variable with a small scope, I think renaming it shorter would be ok:

- [ ] Rename timeSpanDescriptionObj -> descriptionObject